### ISSUE TITLE
Cannot import .lua file from a macro file

### ISSUE THREAD

**Issue by datwaft** [rktjmp/hotpot.nvim#34]
Hello! Sorry for so many issues.

This time I was trying to use another regex library, as a continuation of #33. This library is [LuLPeg](https://github.com/pygy/LuLPeg) and it is a pure Lua port of [LPeg](http://www.inf.puc-rio.br/~roberto/lpeg/); which means that it doesn't need any `.so` files.

When I try to require this file from a macro file it seems to try to compile the file as if it was a `.fnl` file.

Here is the error message:
```log
Error detected while processing /root/.config/nvim/init.lua:
.../packer/start/hotpot.nvim/fnl/hotpot/searcher/module.fnl:104: Compile error in /root/.config/nvim/fnl/core/init.fnl:3
  Parse error in /root/.config/nvim/lua/lulpeg.lua:1
  invalid whitespace after quoting prefix
-- LuLPeg, a pure Lua port of LPeg, Roberto Ierusalimschy's
----------^
stack traceback:
^I[C]: in function 'parse_error'
^I...site/pack/packer/start/hotpot.nvim/lua/hotpot/fennel.lua:3589: in function 'parse_prefix'
^I...site/pack/packer/start/hotpot.nvim/lua/hotpot/fennel.lua:3663: in function '(for generator)'
^I...site/pack/packer/start/hotpot.nvim/lua/hotpot/fennel.lua:2996: in function 'compile-string'
^I...site/pack/packer/start/hotpot.nvim/lua/hotpot/fennel.lua:4014: in function 'loader'
^I...site/pack/packer/start/hotpot.nvim/lua/hotpot/fennel.lua:2004: in function '_2_'
^I...site/pack/packer/start/hotpot.nvim/lua/hotpot/fennel.lua:2007: in function 'require'
^I/root/.config/nvim/fnl/core/macro/macrofile.fnl:1: in function 'loader'
^I...site/pack/packer/start/hotpot.nvim/lua/hotpot/fennel.lua:2041: in function 'require-macros'
^Isrc/fennel/macros.fnl:257: in function <src/fennel/macros.fnl:251>
^I[C]: in function 'xpcall'
^I...
^I...site/pack/packer/start/hotpot.nvim/lua/hotpot/fennel.lua:3000: in function 'compile-string'
^I...te/pack/packer/start/hotpot.nvim/fnl/hotpot/compiler.fnl:23: in function <...te/pack/packer/start/hotpot.nvim/fnl/hotpot/compiler.fnl:22>
^I[C]: in function 'compile_string'
^I...te/pack/packer/start/hotpot.nvim/fnl/hotpot/compiler.fnl:40: in function <...te/pack/packer/start/hotpot.nvim/fnl/hotpot/compiler.fnl:29>
^I[C]: in function 'compile_fnl'
^I.../packer/start/hotpot.nvim/fnl/hotpot/searcher/module.fnl:101: in function <.../packer/start/hotpot.nvim/fnl/hotpot/searcher/module.fnl:89>
^I[C]: in function 'pcall'
^I.../packer/start/hotpot.nvim/fnl/hotpot/searcher/module.fnl:135: in function <.../packer/start/hotpot.nvim/fnl/hotpot/searcher/module.fnl:124>
^I[C]: in function 'require'
^I/root/.config/nvim/init.lua:15: in main chunk
(import-macros {} :core.macro.macrofile)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

The module can be required without any problem in a normal `.fnl` file.

# How to reproduce

Here is the repository to reproduce the issue: https://github.com/datwaft/hotpot-issue_34

**Comment by rktjmp** [comment]
Using #30 fixes this.

**Comment by datwaft** [comment]
> Using #30 fixes this.

Nice! :D

**Comment by rktjmp** [comment]
merged #30
